src/config/staging.ts

```diff
@@ -0,0 +1,11 @@
+import { IConfig } from './types';
+
+const config: IConfig = {
+  env: 'staging',
+  productName: 'Quiz Master',
+  productUrl: 'https://testquiz.quizmaster.com',
+  backendUrl: 'https://api-testquiz.quizmaster.com',
```

Comment:
Do we have this domain?

Reply:
for staging we can use `vercel` app. but for production we may need it